### DISCUSSION TITLE
ratio field Why must one be selected?

### DISCUSSION THREAD

**Discussion by blueMountain007** [payloadcms/payload#2039] [Q&A] [answered]
<img width="935" alt="image" src="https://user-images.githubusercontent.com/16400882/217503227-44698e7f-fa98-4f67-8f58-b2c7a68fd2c5.png">

<img width="425" alt="image" src="https://user-images.githubusercontent.com/16400882/217503306-fc5667be-dad1-4849-b759-360542fe2893.png">

There are cases where neither is selected. This situation cannot be achieved with ratio

**Comment by blueMountain007** [comment]
It can only be realized by select, but it is inconvenient

**Reply by JessRynkar** [reply]
@zhounewz change `require` to `required` and give that a shot.

**Reply by JessRynkar** [reply]
@zhounewz also if your options only need to be true and false, you could use a checkbox field here.

**Comment by blueMountain007** [comment]
Changing from require to required didn't work.
<img width="359" alt="image" src="https://user-images.githubusercontent.com/16400882/217550395-dad5b363-a8bf-4ffd-a040-e14ff2e33c57.png">
There are 3 options for me, unselected, true or false.

**Reply by JessRynkar** [reply]
@zhounewz what version of payload are you on?

**Comment by blueMountain007** [comment]
@JessChowdhury  "payload": "1.2.5"

**Comment by DanRibbens** [comment, marked as answer]
The required validation on radios was fixed in a more recent version. Upgrading Payload will help, but not completely fix your problem.

Radio inputs in the browser don't work the way you want.  There is no way to deselect a value without adding a clear button. You need a third option for 'null' in your list it seems.
